app/assets/javascripts/drafts.coffee

```diff
@@ -76,6 +76,11 @@ $(document).ready ->
     e.stopImmediatePropagation()
 
   incrementElementIndex = (newDiv, multipleIndex, simple, type) ->
```

Comment:
I think there are some elements in the Platforms fields (on the Acquisition Information form) for which the labels are incrementing correctly but the divs for the fields and subfields are not.

For instance, in a new draft on the Acquisition Information form, click on 'Add another Platform' and then for that platform click on 'Add another Instrument'. In the html of the page, you can see a div with `id="draft_platforms_1_instruments"`, and there are divs nested under that, with ids `draft_platforms_1_instruments_0` and `draft_platforms_1_instruments_1`. If we look at the div with `id="draft_platforms_1_instruments_1"`, most of the fields nested there seem to have incremented correctly. But there are two that were not:

1. For Characteristics, there is a label `for="draft_platforms_1_instruments_1_characteristics"`, but the div has `id="draft_platforms_1_instruments_0_characteristics"`. The first child of that div has `id="draft_platforms_1_instruments_0_characteristics_0"`
2. For ComposedOf, the label is `for="draft_platforms_1_instruments_1_composed_of"`, but the div has `id="draft_platforms_1_instruments_0_composed_of"` and the first child also has `id="draft_platforms_1_instruments_0_composed_of_0"`